src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -1139,6 +1143,7 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 		IECore::ConstObjectVectorPtr m_lightShader;
 		std::vector<ArnoldShaderPtr> m_lightFilterShaders;
 		IECore::ConstInternedStringVectorDataPtr m_traceSets;
+		int m_transformType;
```

Comment:
I doubt there's any real performance benefit to storing this as an int rather than an InternedString - we have much bigger fish to fry. But if you really want to do it, we should use the AiEnum API to convert from the string attribute to the internal int, so we'll naturally be in sync with any enum value changes in the future.

Reply:
I'm just nervous because Arnold have explicitly deprecated all functions in their API that take char* instead of their own interned string called AtString, because they are worried about the performance impact of implicit conversions to AtString.  I might try out the AiEnum API approach.